test/WebJobs.Script.Abstractions/BindingMetadataTests.cs

```diff
@@ -21,8 +21,8 @@ public void BindingMetadata_Create_TriggerBinding_Success()
             Assert.Equal("AzureWebJobsStorage", result.Connection);
             Assert.Equal(triggerBinding, result.Raw);
             Assert.Equal(BindingDirection.In, result.Direction);
-            Assert.True(result.Properties.TryGetValue("SupportsDeferredBinding", out bool supportsDeferredBinding));
-            Assert.False(supportsDeferredBinding);
+            Assert.True(result.Properties.TryGetValue("SupportsDeferredBinding", out var supportsDeferredBinding));
```

Comment:
Intrigued by this change. Did this parameter type change?

Reply:
No the type is the same, always has been object - I think maybe the version of the language changed between the two test projects? Was also surprised to see this come up as an error - let me change this to net6.0 and see if this stays the same

Reply:
Ah nope, that didn't do it xD I'm not sure what it could be